src/main/java/org/geppetto/frontend/controllers/WebsocketConnection.java

```diff
@@ -185,6 +188,20 @@ protected void onTextMessage(CharBuffer message) throws IOException
 				connectionHandler.getVersionNumber(requestID);
 				break;
 			}
+			case USER_PRIVILEGES:
+			{
+				connectionHandler.checkUserPriviledges(requestID);
+				break;
+			}
+			case PROJECT_PERSISTENCE_STATE:
```

Comment:
The frontend has a variable: window.Project.persisted.  Can we reuse that instead of creating a new message here?  If so let's remove this guy.